pkg/controller/render/render_controller.go

```diff
@@ -443,6 +443,16 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 		klog.Errorf("Error syncing Generated MCFG: %v", err)
 		return ctrl.syncFailingStatus(pool, err)
 	}
+
+	// TODO: question why not?
+	// imageSets, err := ctrl.imageSetLister.List(selector)
+	// if err != nil {
+	// 	return err
+	// }
+	// sort.SliceStable(imageSets, func(i, j int) bool { return mcs[i].Name < mcs[j].Name })
+	// // if err := ctrl.syncPinnedImageSets(pool, imageSets); err != nil {
```

Comment:
Is this where the controller will populate the pool's spec based on the current PinnedImageSet objects? Or is there somewhere else/will be using some new mechanism? It seems that the MCD will be reading off the spec but I can't see to find that populated anywhere

Reply:
My question is why not reuse the render controller? It would be very simple to add the logic here vs another controller to perform this action.

Reply:
added pinnedimageset controller